app/views/steps/_hidden_inputs_for_calendars.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="hidden--js-enabled">
+  <% 3.times do |i| %>
+    <div class="group">
+      <label for="slot_step_option_<%= i %>">Option <%= i %></label>
+      <select class="SlotPicker-input" name='slots_step[option_<%= i %>]' id="slots_step_<%= i %>">
+        <option value=''>none</option>
+        <% f.object.available_slots.each do |slot| %>
+          <option value="<%= slot.iso8601 %>">
```

Comment:
I wonder if using an `iso8601` value here (and the subsequent changes to the JavaScript) work in some of the older browsers we have to support? Have you tested that?

If not: we should test [all the browser GOV.UK supports](https://www.gov.uk/service-manual/user-centred-design/browsers-and-devices.html) (or at least their oldest versions) before merging this. Gimme a shout tomorrow if you want another pair of hands doing that.